java/integTest/src/test/java/glide/standalone/CommandTests.java

```diff
@@ -394,4 +394,31 @@ public void functionLoad() {
         assertEquals(libName, regularClient.functionLoad(newCode, true).get());
         // TODO test with FCALL
     }
+
+    @Test
+    @SneakyThrows
+    public void copy() {
```

Comment:
we should add this test to cluster/CommandTests.java too.

Reply:
I think since `copy` works in both modes w/o DB and doesnt work in cluster mode with DB, this should only keep the signature with DB and the other ones should go under SharedCommandTests normally